pyproject.toml

```diff
@@ -1,3 +1,90 @@
+# Build-system section
 [build-system]
 requires = ["setuptools>=40.8.0", "wheel"]
 build-backend = "setuptools.build_meta"
+
+# Black section
+[tool.black]
+line-length=80
+check="True"
+diff="True"
```

Comment:
IMO the default for black should be to actually re-format. At least, that's what I would expect when copying the command from the Contributors docs, or using it in a precommit hook. So I suggest to  remove the `check` and `diff` options here and supply them in tox.ini only, as you do for `isort`.

Reply:
Yes, it makes sense.
Will remove them from `pyproject.toml` and move them back in `tox.ini`.